src/main/java/com/chiwanpark/flume/plugins/RedisSubscribeDrivenSource.java

```diff
@@ -68,8 +72,15 @@ public synchronized void start() {
 
     // TODO: consider using Connection Pool.
     jedis = new Jedis(redisHost, redisPort, redisTimeout);
+    if (redisPassword != "") {
```

Comment:
We use 2 spaces to indentation, please change this block to use 2 spaces indentation.